src/connector.js

```diff
@@ -983,8 +983,8 @@ function updateResultListState( newState ) {
 			}
 
 			let breadcrumb = "";
-			let printableUri = stripHtml( result.printableUri );
-			let clickUri = stripHtml( result.clickUri );
+			let printableUri = DOMPurify.sanitize( result.printableUri );
+			let clickUri = DOMPurify.sanitize( result.clickUri );
```

Comment:
Please use `encodeURI()` instead of `DOMPurify.sanitize()` on both `printableUri` and `clickUri`. Also, please move the `.replaceAll( '&' , '&amp;' )` from line 1005 and put it on the variable declaration for `printableUri` at line 986 instead.

Finally, I noticed that if there is no breadcrumb and the code falls back on the full URL, then the `printableUri` is used twice; it should instead be the `clickUri` for the href:
```
breadcrumb = '<p class="location"><cite><a href="' + clickUri + '">' + printableUri + '</a></cite></p>';
```